lib/generators/pundit/policy/templates/policy.rb

```diff
@@ -1,6 +1,12 @@
 <% module_namespacing do -%>
 class <%= class_name %>Policy < ApplicationPolicy
-  class Scope < Scope
+  # NOTE: Prior to Pundit v2.4.0, the inheritance was declared as
+  # `Scope < Scope` rather than `Scope < ApplicationPolicy`. In most cases the
+  # behavior will be identical, but if updating exisiting code, beware of
+  # possible changes to the ancestors:
+  # https://gist.github.com/Burgestrand/4b4bc22f31c8a95c425fc0e30d7ef1f5
```

Comment:
Rather than adding a long explanation as a comment, I thought it better to link to your illustration. If you'd prefer not to have a link to your gist, then maybe there could be a page in the repo?

Reply:
I think that's fine. I'll probably forget about the gist and it'll be there for however long github lets it stay there. The gist also has a comment pointing back to this PR, so if I ever end up in a cleaning mood I'll probably see this and remember to let it be 🙂